core/src/test/scala/unit/kafka/admin/DescribeConsumerGroupTest.scala

```diff
@@ -281,15 +685,27 @@ class DescribeConsumerGroupTest extends KafkaServerTestHarness {
     consumerProps.setProperty("zookeeper.connect", zkConnect)
     oldConsumers += new OldConsumer(Whitelist(topic), consumerProps)
   }
+
+  def addConsumerGroupService(service: ConsumerGroupService): ConsumerGroupService = {
```

Comment:
nit: you could probably just construct the service in this method if you pass the command options.